README.md

```diff
@@ -44,6 +44,21 @@ See Helm `values.yaml`, `enterprise` section for configuration examples.
 > Don't have StackStorm Enterprise License?<br>
 > 90-day free trial can be requested at https://stackstorm.com/#product
 
+## Configuration
+
+The default configuration values for this chart are described in `values.yaml`.
+
+## Ingress
+
+Use an ingress if you need to expose HTTP and HTTPS routes from outside the cluster to services
```

Comment:
Exposing could be done with `LoadBalancer` or with `NodePort` as well. Explanation behind the need of Ingress needs some re-wording.